sktime/forecasting/ets.py

```diff
@@ -238,7 +239,15 @@ def __init__(
         self.n_jobs = n_jobs
 
         super(AutoETS, self).__init__(random_state=random_state)
-
+        
+        if self.auto and (any(param for param in [trend, damped_trend, seasonal]) or error!="add"):
```

Comment:
May be I am missing something, can you please explain this check? What is the goal here - whether user has explicitly passed these 4 parameters, or something else?

Reply:
I'll explain my confusion. If I am not wrong, these parameters will always be present whether or not user explicitly pass the values. If user specifically pass a value, then I felt it's wrong for `auto` to change it without informing user. Otherwise, if user don't pass anything and use the default settings, do we still need to throw a warning?

This may be some design discussion, as I am not familiar how this sort of thing is being handled elsewhere in `sktime`, so cc @fkiraly.

Reply:
I also got confused first, but here is what I think it checks (disjunction of):

* one of `trend`, `seasonal` are not `None`
* `damped_trend=True`
* `error` is not `"add"`

For each of these parameters, that's the condition "is not the default", so it would fire if the user has provided a non-default parameter to any of the affected parameters that `auto` overrides/ignores.

Reply:
This is what I was intending, is there a different way it should be written?

Reply:
(I thought it was fine, just obscure)

Reply:
One minor code suggestion: `any(<list>)` should be enough I think, `any(element for element in <list>)` is probably not required. Please check.

But mainly I have some feedback from user point of view. As a user, if I specify `damped_trend=False` explictly in function call, I expect that to be respected. It may be default value as well, but I want to ensure that it does not get modified because I also pass `auto=True`. If that happens, I'd at least want to be informed.

Now I understand that whether user has passed `False` or it's taken from default will be difficult to check (may be even impossible, I have never tried). But this was my consideration for the feedback.